### PR TITLE
To allow developper to use this bundle with Symfony 3+

### PR DESCRIPTION
**Pull request by frenchcomp**
Update composer to use this bundle with Symfony 3+ and Symfony 2.3+

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,9 +14,9 @@
     ],
     "require": {
         "php": ">=5.4",
-        "symfony/config": "~2.3",
-        "symfony/dependency-injection": "~2.3",
-        "symfony/http-kernel": "~2.3",
+        "symfony/config": "~2.3||~3.2",
```
Comment (meyerbaptiste) [inline composer.json]: `~3.0` is better.

File: spec/CoopTilleuls/Bundle/YmlpBundle/Ymlp/YmlpClientSpec.php [hunk 2]
```diff
@@ -47,7 +49,7 @@ function let(
 
     function it_is_initializable()
     {
-        $this->shouldHaveType('CoopTilleuls\Bundle\YmlpBundle\Ymlp\YmlpClient');
+        $this->shouldHaveType(YmlpClient::class);
```
Comment (meyerbaptiste) [inline spec/CoopTilleuls/Bundle/YmlpBundle/Ymlp/YmlpClientSpec.php]: Can you revert this please? This breaks the support of PHP 5.4. I will update to 5.5 in another PR with the support of Guzzle 6.

File: Ymlp/YmlpClient.php [hunk 3]
```diff
@@ -43,7 +43,7 @@ public function __construct($apiUrl, $apiKey, $apiUsername, ClientInterface $cli
                 'base_url'  => $apiUrl,
                 'defaults'  => [
                     'headers'   => [
-                        'User-Agent'    => 'CoopTilleulsYmlpBundle for Symfony2'
+                        'User-Agent'    => 'CoopTilleulsYmlpBundle for Symfony2/3'
```
Comment (meyerbaptiste) [inline Ymlp/YmlpClient.php]: Simply `for Symfony`?

File: composer.json (hunk 1 above)
Comment (frenchcomp) [inline composer.json]: Done, sorry, for my projects I forbid old non supported version.

File: Ymlp/YmlpClient.php (hunk 3 above)
Comment (frenchcomp) [inline Ymlp/YmlpClient.php]: Done

File: spec/CoopTilleuls/Bundle/YmlpBundle/Ymlp/YmlpClientSpec.php (hunk 2 above)
Comment (frenchcomp) [inline spec/CoopTilleuls/Bundle/YmlpBundle/Ymlp/YmlpClientSpec.php]: It's ok

**Comment by dunglas** [comment]
The README can be updated too:

```
CoopTilleulsYmlpBundle, YMLP API for Symfony2

This bundle integrates the Your Mailing List Provider (YMLP) API into Symfony2 projects.
```

**Comment by meyerbaptiste** [comment]
Can you make the change to the README file @frenchcomp?

**Comment by frenchcomp** [comment]
Ok, i will do it this afternoon

**Comment by frenchcomp** [comment]
Done :)

**Comment by meyerbaptiste** [comment]
Thanks @frenchcomp

**Pull request merged**